sdk/resourcemanager/azure-resourcemanager-samples/src/test/java/com/azure/resourcemanager/samples/GraphRbacTests.java

```diff
@@ -34,7 +34,7 @@ public void testManageUsersGroupsAndRoles() {
             return;
         }
 
-        Assertions.assertTrue(ManageUsersGroupsAndRoles.runSample(authenticated, profile));
+        Assertions.assertTrue(ManageUsersGroupsAndRoles.runSample(azureResourceManager, profile));
```

Comment:
If this changed, we might able to delete a few overload in `TestUtilities` about random rg name and random guid.